app/graphql/types/record_type.rb

```diff
@@ -108,9 +108,14 @@ class AlternateTitleType < Types::BaseObject
 
   # Warning: related_place was supposed to be an array but was incorrectly a string in graphql for v1
   class LocationType < Types::BaseObject
+    field :geopoint, String, deprecation_reason: 'Use `geoshape`'
     field :geoshape, String, description: 'GeoShape data for the location, if applicable'
```

Comment:
We may want to provide more details on what is in this field. I believe dataeng is consistently (we should confirm) using the BBOX option for the envelope as described here:
https://opensearch.org/docs/latest/field-types/supported-field-types/geo-shape/#envelope

If that turns out to be true, noting something like this for description would probably be helpful.
'GeoShape data for the location when available in WKT format. i.e. `BBOX (minLon, maxLon, maxLat, minLat)`'